src/dotnet/Program.cs

```diff
@@ -114,6 +114,8 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 }
                 else
                 {
+                    firstTimeUseSentinelPath = ConfigureDotNetForFirstTimeUse();
```

Comment:
Looking through the code, we are making a daisy-chain affect with our environment variables.  If I turn off "DOTNET_SKIP_FIRST_TIME_EXPERIENCE", then Telemetry is turned off.  Is this what we want?